problems/data_science/chipmunks/requirements.txt

```diff
@@ -0,0 +1,7 @@
+numpy
```

Comment:
It would be nice to version lock the libraries. I was getting some errors on 3.7.2, but no issues on 3.6.1.
In the interest of time lets go forward with it, but if you could submit a followup pr that would be very helpful.